### PR TITLE
APEXMALHAR-2431 Create Kinesis Input operator which emits byte array

### PR DESCRIPTION
**Pull request by deepak-narkhede**

### CODE REVIEW

**Comment by deepak-narkhede** [comment]
@chaithu14 . Please review.

File: contrib/src/main/java/com/datatorrent/contrib/kinesis/KinesisByteArrayInputOperator.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datatorrent.contrib.kinesis;
+
+import java.nio.ByteBuffer;
+
+import com.amazonaws.services.kinesis.model.Record;
+
+public class KinesisByteArrayInputOperator extends AbstractKinesisInputOperator<byte[]>
```
Comment (chaithu14) [inline contrib/src/main/java/com/datatorrent/contrib/kinesis/KinesisByteArrayInputOperator.java]: Please add java doc.

File: contrib/src/test/java/com/datatorrent/contrib/kinesis/KinesisInputOperatorTest.java [hunk 2]
```diff
@@ -182,6 +182,56 @@ public void testKinesisInputOperator() throws Exception
...
+    KinesisConsumer consumer = new KinesisConsumer();
+    consumer.setStreamName(streamName);
+    consumer.setRecordsLimit(totalCount);
+    node.setConsumer(consumer);
+
+    // Create Test tuple collector
+    CollectorModule<byte[]> collector = dag.addOperator("TestMessageCollector", new CollectorModule<byte[]>());
+
+    // Connect ports
+    //dag.setOutputPortAttribute(node.outputPort, PortContext.TUPLE_CLASS, Byte.class);
```
Comment (chaithu14) [inline contrib/src/test/java/com/datatorrent/contrib/kinesis/KinesisInputOperatorTest.java]: Remove the line, if not required.

File: contrib/src/main/java/com/datatorrent/contrib/kinesis/KinesisByteArrayInputOperator.java (hunk 1 above)
Comment (deepak-narkhede) [reply contrib/src/main/java/com/datatorrent/contrib/kinesis/KinesisByteArrayInputOperator.java]: @chaithu14 . Added java doc. Please have a look.

File: contrib/src/test/java/com/datatorrent/contrib/kinesis/KinesisInputOperatorTest.java (hunk 2 above)
Comment (deepak-narkhede) [reply contrib/src/test/java/com/datatorrent/contrib/kinesis/KinesisInputOperatorTest.java]: Done. Thanks.

**Comment by chaithu14** [comment]
Looks good to me.

**Review by chaithu14 — APPROVED** [review]

**Pull request merged**